src/Codeception/Configuration.php

```diff
@@ -570,7 +570,22 @@ public static function isEmpty()
      */
     public static function append(array $config = [])
     {
-        return self::$config = self::mergeConfigs(self::$config, $config);
+        self::$config = self::mergeConfigs(self::$config, $config);
+        
+        if (isset(self::$config['paths']['log'])) {
```

Comment:
Should, but without them some checks were not successful ... probably in some test cases are not defined all paths (see first commit checks)